src/modules/core/components/Lightbox.js

```diff
@@ -1,10 +1,6 @@
 import React from "react";
-import { bindActionCreators } from "redux";
-import { connect } from "react-redux";
 import injectJSS from "react-jss";
```

Comment:
Ehh, minor detail but the HOC name is either `injectSheet` or `withStyles` (actually `withStyles` is the official name I think)